### ISSUE TITLE
Rank-guided learning accelerates automated enzyme engineering

### ISSUE THREAD

**Issue by delalamo** [delalamo/SKM#640]
```bibtex
@article{xu2026,
  title = {Rank-guided learning accelerates automated enzyme engineering},
  author = {Xu, Jingyi and Zheng, Yan and Sundarraj, Rajamanikandan and Woycechowsky, Kenneth and Yuchi, Zhiguang and Yuan, Yingjin},
  year = {2026},
  journal = {Nature Communications},
  publisher = {Springer Science and Business Media LLC},
  doi = {10.1038/s41467-026-76264-2},
  url = {https://doi.org/10.1038/s41467-026-76264-2}
}
```

**Comment by delalamo** [comment]
#### Candidate atomic notes

- **Combining pairwise ranking with pointwise regression improves supervised protein fitness prediction** [@xu2026]. Main Figure 2B–D shows higher Spearman correlation and lower standardized MSE than matched pointwise-loss baselines across 212 held-out ProteinGym DMS datasets; Figure 2E shows parity with EVOLVEpro in the smallest few-shot setting and increasing advantages as labeled data accumulate. Evidence is strong from a matched benchmark with shared embeddings, architecture, splits, and training protocol. The comparison is not exhaustive across contemporary methods, and global Spearman/MSE may not measure top-hit prioritization. [[Fitness prediction]] discusses evaluation metrics and low-data behavior but not hybrid loss design. Related: [[Fitness prediction]]; [[Precision decreases and recall increases as variant effect prediction focuses on top-performing point mutations]]; [[Using evolutionary information for label smoothing during inverse folding training leads to improved fitness prediction]]. Placement: new atomic note, `Pairwise ranking and pointwise regression jointly improve protein fitness prediction.md`.

- **Iterative experimental feedback progressively enriches high-activity variants during model-guided enzyme engineering** [@xu2026]. Main Figure 3A–C and Figure 6A–C show progressively improving activity distributions across two enzyme campaigns, culminating in reported gains of 57-fold and approximately 104-fold; Figure 4A–E and Supplementary Figure 8 show prediction precision increasing from 42.6% to 63.4%, while Figure 4F provides an independent saturation benchmark in which improved variants at 12 of 27 model-nominated sites exceeded the literature-derived reference. Evidence comes from two enzyme case studies with highly reproducible replicate measurements and an independent validation set. There is no controlled ablation separating model updating, candidate selection, and automation; generalization is limited to two tasks, rapid high-fidelity assays remain necessary, optimization was single-objective, and the reporting summary records no randomization, blinding, or sample-size calculation. This supports [[Accurate fitness landscapes are unnecessary for productively engineering enzymes or proteins]], although it does not directly test that note’s stronger claim about landscape accuracy. Related: [[Directed evolution]]; [[Accurate fitness landscapes are unnecessary for productively engineering enzymes or proteins]]; [[Focused protein sequence libraries are poor training sets]]. Placement: new atomic note, `Iterative experimental feedback improves model-guided enzyme engineering.md`.

- **Additive models remain competitive for higher-order variant ranking even when guided engineering enriches positive epistasis** [@xu2026]. Main Figure 5A and Figure 6F report increasing positive-epistasis fractions with mutational order among selected combinatorial variants, whereas Supplementary Table 6 gives the additive baseline the highest across-dataset mean Spearman correlation across 12 higher-order-variant datasets (0.6380 versus 0.5995 for RankReg and 0.5949 for MULTI-evolve), with the leading method varying by dataset. Evidence pairs direct combinatorial measurements with a broader benchmark containing public DMS datasets and both study systems. The campaign variants were deliberately selected rather than sampled representatively, model results were averaged across splits while the additive baseline was deterministic, and aggregate Spearman correlation may miss rare top-performing candidates. This supports the additive-model evidence in [[Epistasis is rare during evolution]], while qualifying any absolute reading because selected higher-order candidates were enriched for beneficial non-additivity. Related: [[Epistasis is rare during evolution]]; [[ML improves directed evolution when navigating fitness landscapes with greater non-magnitude epistasis]]; [[Fitness landscapes are locally smooth but globally rugged]]; [[Second-shell mutations can yield positive epistasis]]. Placement: append under `#### Details` in [[Epistasis is rare during evolution]].

- **Model-guided enzyme engineering can identify beneficial variants that are rare in natural sequence alignments** [@xu2026]. Supplementary Figure 7A–B shows that several beneficial changes observed in the first enzyme system occur at frequencies below 1% among homologs; Main Figure 4F–G independently validates activity-enhancing model nominations across both catalytic-pocket and distal regions. Evidence is supported by alignment analysis and a validation set excluded from model training and candidate selection. The result was demonstrated in one enzyme system, rarity does not prove that evolutionary models would reject a variant, and the study does not isolate whether iterative feedback was necessary for discovery. This supports [[Directed evolution can uncover beneficial sequence modifications that might be missed by inverse folding]], but the present study evaluates a PLM-guided closed loop rather than inverse folding directly. Related: [[Directed evolution can uncover beneficial sequence modifications that might be missed by inverse folding]]; [[Natural sequences deviate from highest stability variants]]; [[Directed evolution]]. Placement: new atomic note, `Model-guided enzyme engineering identifies beneficial variants that are rare in natural sequence alignments.md`.

#### Metadata check

The issue metadata—title, author list, journal, year, and DOI `10.1038/s41467-026-76264-2`—matches the publisher record. The manuscript is currently labeled an unedited Article in Press, so volume, issue, and article-number fields should not be invented. Neither the DOI, title, nor citekey `xu2026` currently appears in `bibliography.bib`. The complete accepted manuscript, Supplementary Information, supplementary-file index, and reporting summary were reviewed; operational supplementary datasets were intentionally not summarized.
